src/server.rs

```diff
@@ -76,39 +150,60 @@ pub struct Server<L, M>
 
     /// Currently registered reconnection timeouts.
     reconnection_timeouts: HashMap<Token, TimeoutHandle>,
+
+    /// Configured timeouts
+    timeout_config: TimeoutConfiguration,
 }
 
 /// The implementation of the Server.
 impl<L, M> Server<L, M>
     where L: Log,
           M: StateMachine
 {
+    pub fn new(
```

Comment:
The fact that both a `create` and `new` exists confuses me. Is there a reason for this?